source/index.js

```diff
@@ -2,15 +2,22 @@ var Pixi = require("pixi.js")
 var Afloop = require("afloop")
 var Keyb = require("keyb")
 
-var WIDTH = 230, HEIGHT = 130
+import Player from "./player.js"
```

Comment:
Nice! I need to read up on my ECMA 7; this is the new cool way of importing introduced in that new spec. I haven't used it before! Does it work?